lib/tree-ignore.js

```diff
@@ -87,23 +87,32 @@ _fUpdate = function() {
 
     oIgnore = ignore().addIgnoreFile( _oAtomIgnoreFile.getPath() );
 
+    sProjectRoot = "";
+    aTreeEntries = [];
```

Comment:
Remove this line, it is a remnant of some removed code.
I wonder why linter didn't caught it. Isn't IE6 supposed to have "use strict" by default?